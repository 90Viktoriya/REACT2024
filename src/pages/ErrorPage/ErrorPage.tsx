import { ComponentsCaptions } from '../../data/ComponentsCaptions';

export function ErrorPage() {
  return <h1>{ComponentsCaptions.ERROR_PAGE}</h1>;
}
