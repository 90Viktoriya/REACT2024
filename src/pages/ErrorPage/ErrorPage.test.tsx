import { render } from '@testing-library/react';
import { ErrorPage } from './ErrorPage';

describe('Given List component', () => {
  it('When rendered, should match snapshot', () => {
    const { asFragment } = render(<ErrorPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
