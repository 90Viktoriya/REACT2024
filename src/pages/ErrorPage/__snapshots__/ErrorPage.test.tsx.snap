// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given List component > When rendered, should match snapshot 1`] = `
<DocumentFragment>
  <h1>
    OOPS! Page not found
  </h1>
</DocumentFragment>
`;
